### PR TITLE
Add description for podman/buildah_testsuite

### PR DESCRIPTION
**Pull request by ricardobranco777**
Add description for podman/buildah_testsuite

### CODE REVIEW

File: job_groups/opensuse_tumbleweed.yaml [hunk 1]
```diff
@@ -323,6 +323,8 @@ scenarios:
             CONTAINER_RUNTIMES: 'podman'
             K3S_INSTALL_UPSTREAM: '1'
       - containers_host_podman_testsuite:
+          description: |-
+            Maintainer: qe-c@suse.de https://confluence.suse.com/display/qasle/podman+upstream+tests
```
Comment (Vogtinator) [inline job_groups/opensuse_tumbleweed.yaml:327]: It's not that great having an internal URL in a public description

Comment (ricardobranco777) [reply job_groups/opensuse_tumbleweed.yaml:327]: > It's not that great having an internal URL in a public description

True, but what's the alternative?

Comment (Vogtinator) [reply job_groups/opensuse_tumbleweed.yaml:327]: New page on https://progress.opensuse.org/projects/openqatests/wiki?

Comment (ricardobranco777) [reply job_groups/opensuse_tumbleweed.yaml:327]: > New page on https://progress.opensuse.org/projects/openqatests/wiki?

The problem is that confidential SUSE information could spill there.  I haven't even finished the documentation and don't want to be in paranoid mode while doing it. =/

Comment (grisu48) [reply job_groups/opensuse_tumbleweed.yaml:327]: I do a agree with @Vogtinator. openSUSE related job groups and documentation should be open from the beginning.

**Review by grisu48 — APPROVED** [review]

**Comment by pdostal** [comment]
I hope it's ok for you @Vogtinator to merge this.

**Comment by Vogtinator** [comment]
> I hope it's ok for you @Vogtinator to merge this.

Yes, if it's the currently best option.

**Pull request merged**